README.md

```diff
@@ -1,79 +1,135 @@
+```json
+{
+  "filter": {},
+  "fields": [
+    "string"
+  ],
+  "format": "string",
+  "feature": "string"
+}
+```
+### Filter
+
+To select cells, the API supports a simple yet expressive language for specifying complex metadata and expression data
+filters capable of representing nested AND/OR structures as a JSON object. There are two types of filter objects to
+achieve this:
+
+#### Comparison filter
+```
+{
+  "op": one of [ =, !=, >, <, >=, <=, in ],
+  "field": a metadata filter,
+  "value": string or int or list
+}
+```
+
+#### Logical filter
+```
+{
+  "op": one of [ and, or, not ],
+  "value": array of 2 filter objects if op==and|or, filter object if op==not
+}
+```
+
+These filter types can be recursively nested via the `value` field of a logical filter.
 
-#### File Converter
+#### Filter examples
 
-A file conversion job deployed on AWS Batch is used to support multiple output [file formats](#file-formats). This job
-converts ``.zarr`` expression matrices to the desired file format and writes the result to S3.
+Select all full length cells:
+```
+...
+  "filter": {
+    "op": ">=",
+    "value": "full length",
+    "field": "library_preparation_protocol.end_bias"
+  }
+...
+```
 
-#### DynamoDB
+Select all cells from the "Single cell transcriptome analysis of human pancreas" project with at least 3000 genes
+detected:
+```
+...
+  "filter": {
+    "op": "and",
+    "value": [
+      {
+        "op": "=",
+        "value": "Single cell transcriptome analysis of human pancreas",
+        "field": "project.project_core.project_short_name"},
+      {
+        "op": ">=",
+        "value": 3000,
+        "field": "genes_detected"
+      }
+    ]
+  }
+...
+```
 
-DynamoDB tables are used to track the state and progress of a request. The following is a description of the tables:
+The list of available filter names is available at `/v1/filters`. To retrieve more information about a specific filter,
+GET `/v1/filters/<filter>`.
 
-| **Table name** | **Description** |
-|---|---|
-| Cache table | Caches requests by a hash of its input parameters. |
-| State table | Tracks the progress of a request. |
-| Output table | Stores output values of the request (e.g. file format, errors). |
-| Lock table | Manages locks for across distributed nodes. |
+### Fields
 
+Users can specify a list of metadata fields to be exported with an expression matrix. The list of available metadata
+fields is available at `/v1/fields`. More information about a specific field is available at
+`/v1/fields/<field>`.
 
-### File formats
+### Format
 
-The DCP MS enables users to prepare expression matrices in several formats by supplying the `format` parameter in the
-POST request to the `/matrix` endpoint. The following is a list of supported file formats:
+The Matrix Service supports generating matrices in the following 3 formats:
 
-- [.zarr](https://zarr.readthedocs.io/en/stable/) (default)
-- [.loom](http://loompy.org/)
+- [.loom](http://loompy.org/) (default)
```

Comment:
Should we remove the leading `.`s here? I wonder if people will think you're supposed to include them in the request body.